api/src/test/java/io/opentelemetry/trace/TraceStateTest.java

```diff
@@ -52,7 +52,7 @@ void getEntries() {
     assertThat(secondTraceState.getEntries())
         .containsExactly(TraceState.Entry.create(SECOND_KEY, SECOND_VALUE));
     assertThat(multiValueTraceState.getEntries())
-        .containsExactly(
+        .containsExactlyInAnyOrder(
```

Comment:
isn't the order actually important here?

Reply:
Confirmed that the order seems to be changing here - filed #1499 will look into it in a follow up.

Reply:
I think the order is supposed to be the opposite of what you put in, if you look at the javadoc on the TraceState.Builder class. I think this test just wasn't asserting that fact.

Reply:
Ah you did the digging for me! Thanks I went ahead and updated the test here.

Reply:
I do like that assertJ makes it super clear when you aren't enforcing the order, though!